src/Data/Fin/Properties.agda

```diff
@@ -22,6 +22,7 @@ open import Data.Product using (∃; ∃₂; ∄; _×_; _,_; map; proj₁; uncur
 open import Data.Sum.Base as Sum using (_⊎_; inj₁; inj₂; [_,_])
 open import Data.Sum.Properties using ([,]-map-commute; [,]-∘-distr)
 open import Function.Base using (_∘_; id; _$_)
+open import Function.Bundles using (_↔_; mk↔′; Inverse)
```

Comment:
No need to import `Inverse`

Reply:
Probably. I think this reflects an intermediate state where I needed it to convince Agda of something.